DuckDuckGo/TabViewController.swift

```diff
@@ -174,6 +175,7 @@ class TabViewController: UIViewController {
         didSet {
             updateTabModel()
             delegate?.tabLoadingStateDidChange(tab: self)
+            historyCapture.titleDidChange(title, forURL: url)
```

Comment:
For whatever reason, when I first started testing this by visiting `nytimes.com` I kept getting visits with no title. After I added some logging, I can't reproduce the issue even once. 🤷 Mentioning it here in case it sounds familiar, but if not then let's just ignore it.

Reply:
I'll see if I can replicate on a clean build.